ingest/ingestHelpers.py

```diff
@@ -1,20 +1,15 @@
 __author__ = 'Hao'
```

Comment:
What is this file?  This has a dataset specific-usage of Ingest yet is called ingestHelpers?  You should probably have the ingest-datasets script contain a class DatasetIngest which extends Ingest and includes the dataset-specific logic.